src/utilcode/clrhost_nodependencies.cpp

```diff
@@ -383,6 +394,10 @@ FastFreeInProcessHeapFunc __ClrFreeInProcessHeap = (FastFreeInProcessHeapFunc) C
 
 const NoThrow nothrow = { 0 };
 
+#ifdef HAS_ADDRESS_SANITIZER
+// use standard heap functions for address santizier
```

Comment:
I think it is ok as long as it applies to sanitizer builds only ... I suspect that the sanitizer depends on its own implementations of these operators to be used.